_overviews/scala3-book/where-next.md

```diff
@@ -2,6 +2,7 @@
 title: Where To Go Next
 type: chapter
 description: Where to go next after reading the Scala Book
+languages: [zh-cn]
```

Comment:
it seems that this page has not been translated, so the language picker is broken

Reply:
add file.